core/common/src/main/java/alluxio/wire/Scope.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
```

Comment:
@aaudiber , do we have a convention to put all classes with `fromThrift` and `toThrift` to package `alluxio.wire`? If yes, I don't feel this is actually necessary and we loss a bit information indicated from original package name in this way.

Reply:
Recently, the thriftUtils file which was responsible to contain all the fromThrift and toThrift functions was deleted. All the fromThrift and toThrift funtions were put back to the wire files. Maybe because they think thriftUtils just provide a wrapper and actually do not do anything else.

Reply:
We moved all wire types to the `wire` package in https://github.com/Alluxio/alluxio/pull/2652, and afaik we've been following the convention since then. Any class that gets converted to/from a thrift type is a wire type.

What do you think of having sub-packages within `wire` so that we can keep the information, e.g. put `Scope` in `alluxio.wire.configuration`

If we want to drop the convention of putting all wire types in `wire`, we should move all the types currently in `wire` to a more appropriate package. For the purpose of this PR I'd like to stick with the convention or amend it slightly to allow sub-packages under `alluxio.wire`

Reply:
SG